website/docs/installation/documentdb/docker.md

```diff
@@ -60,10 +60,88 @@ See [Set up PostgreSQL connection](../../security/authentication.md#set-up-postg
 
 3. See [FerretDB Docker installation](../ferretdb/docker.md) for more details on connecting to FerretDB.
 
-## Development image for PostgreSQL with DocumentDB extension
+### Run development image
 
 The development image for PostgreSQL with DocumentDB extension
 [`ghcr.io/ferretdb/postgres-documentdb-dev`](https://ghcr.io/ferretdb/postgres-documentdb-dev)
 is recommended for debugging problems.
 It includes additional debugging features that make it significantly slower.
 For this reason, it is not recommended for production use.
+
+## Updating to a new DocumentDB release
+
+Before [updating to a new FerretDB release](../ferretdb/docker.md#updating-to-a-new-ferretdb-release), it is critical to install the matching DocumentDB image first.
+
+The following steps are critical to ensuring a successful update.
+
+Edit your Compose file to use the matching DocumentDB image tag.
+You can find the correct tag in the DocumentDB release notes (for example: `17-0.103.0-ferretdb-2.2.0`).
+Then run:
+
+```sh
+docker compose pull <documentdb-container-name>
+docker compose up -d <documentdb-container-name>
+```
+
+Next, from within your `postgres` database, upgrade the DocumentDB extension by running:
+
+```sh
+
+docker compose exec <documentdb-container-name> \
+  psql -U <username> -d postgres -c 'ALTER EXTENSION documentdb UPDATE;'
+```
+
+Replace `<documentdb-container-name>`, `<username>`, `<password>`, and `<host>` as needed.
+
+After the extension update, update `postgresql.conf` settings using a mounted config to ensure the following configurations are present.
+First, create a local file (e.g., `postgresql.custom.conf`) with the following content:
+
+```text
+shared_preload_libraries = 'pg_cron,pg_documentdb_core,pg_documentdb'
+cron.database_name       = 'postgres'
+
+documentdb.enableLetAndCollationForQueryMatch = true
+documentdb.enableNowSystemVariable            = true
+documentdb.enableSortbyIdPushDownToPrimaryKey = true
+
+documentdb.enableSchemaValidation             = true
+documentdb.enableBypassDocumentValidation     = true
+
+documentdb.enableUserCrud                     = true
+documentdb.maxUserLimit                       = 100
+```
+
+Then, mount the config in the DocumentDB container of your Compose file:
```

Comment:
That completely overrides the existing config, not adds to it. We should not do that.